Builds with the `tracing` feature are broken

**To Reproduce:**

STR:
- build with `./mach build --features=tracing`

Expected:
- no errors

Observed:
- build error due to changes from a433b202595 :
```
error[E0599]: no method named `get_url` found for reference `&<D as bindings::codegen::DomTypes::DomTypes>::GlobalScope` in the current scope
  --> components/script/dom/bindings/settings_stack.rs:74:34
   |
74 |                     url = global.get_url().to_string(),
   |                                  ^^^^^^^ method not found in `&<D as DomTypes>::GlobalScope`
   |
``` 

cc @jdm

--- a/components/script/dom/globalscope.rs
+++ b/components/script/dom/globalscope.rs
@@ -3352,6 +3352,8 @@ pub(crate) trait GlobalScopeHelpers<D: crate::DomTypes> {
     fn incumbent() -> Option<DomRoot<D::GlobalScope>>;
 
     fn perform_a_microtask_checkpoint(&self, can_gc: CanGc);
+
+    fn get_url(&self) -> ServoUrl;
 }
 
 #[allow(unsafe_code)]
@@ -3386,5 +3388,9 @@ impl GlobalScopeHelpers<crate::DomTypeHolder> for GlobalScope {
 
     fn perform_a_microtask_checkpoint(&self, can_gc: CanGc) {
         GlobalScope::perform_a_microtask_checkpoint(self, can_gc)
+    }
+
+    fn get_url(&self) -> ServoUrl {
+        self.get_url()
     }
 }

